bff/src/Bff/Extensions/HttpContextExtensions.cs

```diff
@@ -11,6 +12,20 @@ namespace Duende.Bff;
 
 internal static class HttpContextExtensions
 {
+    public static void ReturnHttpProblem(this HttpContext context, string title, params (string key, string[] values)[] errors)
+    {
+        var problem = new HttpValidationProblemDetails()
+        {
+            Status = (int)HttpStatusCode.BadRequest,
+            Title = title,
+            Errors = errors.ToDictionary()
+        };
+        context.Response.StatusCode = problem.Status.Value;
+        context.Response.ContentType = "application/problem+json";
```

Comment:
Are you intending to use problem details broadly in the BFF?

Reply:
Yes, but just to be sure I've put up a discussion point in the chat channel.